src/components/sbb-slider/sbb-slider.tsx

```diff
@@ -0,0 +1,197 @@
+      max = +this.max;
+      value =
+        this.value && this.value !== ''
+          ? +this.value
+          : this.valueAsNumber
+          ? this.valueAsNumber
+          : +this.min + (+this.max - +this.min) / 2;
+    }
+    const mathFraction: number = (value - min) / (max - min);
+    this._valueFraction =
+      isNaN(mathFraction) || mathFraction < 0 ? 0 : mathFraction > 1 ? 1 : mathFraction;
+  }
+
+  /** Emits the change event. */
+  private _emitChange(): void {
+    this.sbbChange.emit({
+      value: this._rangeInput.valueAsNumber,
+      max: +this._rangeInput.max,
+      min: +this._rangeInput.min,
+    });
+  }
+
+  public render(): JSX.Element {
+    const inputAttributes = {
+      name: this.name || null,
+      form: this.form || null,
+      min: this.min || null,
+      max: this.max || null,
+      disabled: this.disabled || this.readonly || null,
+      valueAsNumber: this.valueAsNumber || null,
+      value: this.value || null,
+      'aria-label': this.accessibilityLabel || null,
+      'aria-describedby': this.accessibilityDescribedby || null,
+      'aria-labelledby': this.accessibilityLabelledby || null,
+    };
+
+    return (
+      <Host class={{ 'sbb-form-field-element': this._isInFormField }}>
+        <div class="sbb-slider__wrapper">
+          {this.startIcon && (
+            <slot name="prefix">
+              <sbb-icon name={this.startIcon} />
+            </slot>
+          )}
+
+          <div
+            class="sbb-slider__container"
+            style={{
+              '--sbb-slider-value-fraction': this._valueFraction.toString(),
+            }}
+          >
+            <input
+              class="sbb-slider__range-input"
+              type="range"
+              {...inputAttributes}
+              onChange={() => this._emitChange()}
+              onInput={() => this._handleChange()}
+              ref={(input) => (this._rangeInput = input)}
+            />
+            <div
+              class={{
+                'sbb-slider__line': !this.disabled && !this.readonly,
+                'sbb-slider__line-disabled': this.disabled,
+                'sbb-slider__line-readonly': this.readonly,
+              }}
+            >
+              <div
+                class={{
+                  'sbb-slider__selected-line': !this.disabled && !this.readonly,
+                  'sbb-slider__selected-line-disabled': this.disabled,
+                  'sbb-slider__selected-line-readonly': this.readonly,
+                }}
+              ></div>
+            </div>
+            <div
+              class={{
+                'sbb-slider__knob': !this.disabled && !this.readonly,
+                'sbb-slider__knob-disabled': this.disabled,
+                'sbb-slider__knob-readonly': this.readonly,
+              }}
+            ></div>
+          </div>
+          {this.endIcon && (
+            <slot name="suffix">
+              <sbb-icon name={this.endIcon} />
+            </slot>
+          )}
```

Comment:
The slot should always be rendered.
```suggestion
          <slot name="suffix">
            {this.endIcon&& <sbb-icon name={this.endIcon} />}
          </slot>
```